src/mbgl/renderer/render_orchestrator.cpp

```diff
@@ -144,7 +144,12 @@ RenderOrchestrator::~RenderOrchestrator() {
     // to the scheduler because it cannot be destroyed from one of its own pool threads.
     constexpr auto deferredCleanupTimeout = Milliseconds{1000};
     [[maybe_unused]] const auto remaining = threadPool->waitForEmpty(deferredCleanupTimeout);
+// this assert is causing Android Instrumentation tests to fail
+// since they need to run in on a debug build
+// ignore it for now, see issue https://github.com/maplibre/maplibre-native/issues/2187
```

Comment:
Nice to see we have a backlog task to check this out

Reply:
Marc already has a PR open.